src/ui/components/MetadataCard/index.js

```diff
@@ -25,10 +25,13 @@ const MetadataCard = (props: Props) => {
           throw new Error('title is required');
         }
 
+        const hasContent = content || content === '' ||
```

Comment:
What about this?

```js
// <some comment here explaining '0' strings>
const hasContent = content || typeof content === 'string';
```

Using `parseInt()` to check for string numbers seems awkward.

Reply:
It deserves a comment indeed because I do not check string numbers with `parseInt()`. Numbers are allowed but `0` is interpreted as `false` here..

Reply:
I simplified with a test `content === 0`, it is more obvious I think.